source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -1,181 +1,191 @@
 ^^^^^^^^^^^^^^^
 
-- Holonomic Drive Controller
-  - A controller that teams with holonomic drivetrains (i.e. swerve and mecanum) can use to follow trajectories. This also supports custom ``Rotation2d`` heading inputs that are separate from the trajectory because heading dynamics are decoupled from translational movement in holonomic drivetrains.
-
-- Added support for scheduling functions more often than the robot loop via ``addPeriodic()`` in TimedRobot. Previously, teams had to make a Notifier to run feedback controllers more often than the TimedRobot loop period of 20ms (running TimedRobot more often than this is not advised). Now, users can run feedback controllers more often than the main robot loop, but synchronously with the TimedRobot periodic functions so there aren't any thread safety issues. See an example :ref:`here <docs/software/convenience-features/scheduling-functions:Scheduling Functions at Custom Frequencies>`.
-
-- Added a ``toggle()`` function to Solenoid and DoubleSolenoid.
-
-- Added a ``SpeedControllerGroup`` constructor that takes a ``std::vector<>`` (C++) / ``SpeedController[]`` (Java), allowing the list to be constructed dynamically. (Teams shouldn't use this directly. This is only intended for bindings in languages like Python.)
 
-- Added methods (``isOperatorControlEnabled()`` and ``isAutonomousEnabled()``) to check game and enabled state together.
+- Rewrite Mechanism2d
+- Various ``getInstance()`` functions have been deprecated and replaced with static functions.
+- Add support for DMA to Java
+- ``Counter`` has been deprecated. Please use ``ExternalDirectionCounter``, ``Tach``, ``UpDownCounter``
+- ``Encoder`` has been deprecated. Please use ``QuadratureEncoder`` instead
+- Added C++ TankDrive example
+- GenericHID has been updated to static functions for non-defined controller types
+- Added ``DS4Controller`` controller class
 
-- Added a ``ScopedTracer`` class for C++ teams to be able to time pieces of code. Simply instantiate the ``ScopedTracer`` at the top of a block of code and the time will be printed to the console when the instance goes out of scope.
-
-- Added a static method ``fromHSV(int h, int s, int v)`` to create a ``Color`` instance from HSV values.
+Breaking Changes
+^^^^^^^^^^^^^^^^
 
-- Added RT priority constructor to ``Notifier`` in C++. This makes the thread backing the Notifier run at real-time priority, reducing timing jitter.
+- ``PDP`` is now ``PowerDistribution``
+- Various ``PCM`` related classes (Solenoid, Compressor) now require a ``PneumaticsModuleType``
+- Sendable has been broken up into NT and non-NT components. This will make it easier for vendors and external libraries to integrate sendable.
+- ``wpi::ArrayRef`` has been replaced with ``wpi::span``
+- ``InterruptableSendableBase`` has been broken up
+  - ``AsynchronousInterrupt``
+  - ``SynchronousInterrupt``
+- ``DriverStation.reportWarning`` and ``DriverStation.reportError`` have been removed. See the ``Errors.h`` header
```

Comment:
If this is C++ only, I'd note that and capitalize the `Report` functions. If this also affects Java, what's the replacement?